src/theme.rs

```diff
@@ -555,6 +556,35 @@ pub trait Theme {
         parts.push("".into());
         parts.join("\n")
     }
+
+    /// Returns a multiline text rendering to be used with prompts. This function
+    /// adds the left-bar to all lines but the first, encompases the remainder of the
+    /// message (including new-lines) with the side-bar, and finally ends with
+    /// the section end character.
+    fn format_multiline_text(&self, text: &str) -> String {
+        let lines: Vec<_> = text.lines().collect();
+        let mut parts: Vec<String> = Vec::new();
+
+        for (i, line) in lines.iter().enumerate() {
+            let formatted_line = match i {
+                0 => line.to_string(),
+                _ if i < lines.len() - 1 => format!(
+                    "{bar}  {line}",
+                    bar = self.bar_color(&ThemeState::Active).apply_to(S_BAR),
+                    line = line
+                ),
+                _ => format!(
+                    "{bar}  {line}\n{end}",
+                    bar = self.bar_color(&ThemeState::Active).apply_to(S_BAR),
+                    end = self.bar_color(&ThemeState::Active).apply_to(S_BAR_END),
+                    line = line
+                ),
+            };
```

Comment:
Instead of copy-pasting the whole `format!` twice, you can define `{bar}  {line}{end}`, where `end` is either empty "", or "\n{S_BAR_END}".